pkg/ingester/activeseries/custom_trackers_config.go

```diff
@@ -11,6 +11,8 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const maxNumberOfTrackers = 64000
```

Comment:
This should be `const maxNumberOfTrackers = math.MaxUint16` (not posting the suggestion because an import for that is missing), and I'd add a comment stating why: because we use uint16 in `PreAllocDynamicSlice`.